docs/upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -79,4 +79,10 @@ There you can find links to upgrade notes for other versions too.
         +       arguments:
         +           $useInlineEditation: false
         ```
+- use `CronModuleExecutor` as a service in DIC ([#1314](https://github.com/shopsys/shopsys/pull/1314))
```

Comment:
Seeing that only `protected` attributes of the class have changed, maybe it would be helpful to mention that these notes apply only when you extend the facade:

```suggestion
- use `CronModuleExecutor` as a service in DIC ([#1314](https://github.com/shopsys/shopsys/pull/1314))
    - if you haven't extended `CronFacade` class in your project you don't have to do anything else during the upgrade
```